modules/developers-guide/pages/default-wallet.adoc

```diff
@@ -17,54 +17,206 @@ Because only canisters require and consume cycles—to perform operations and to
+
+* Access the cycles wallet balance and all other wallet-related information.
+* Send cycles to other canisters.
+* Receive cycles from other canisters.
+* Act as the "message-caller" principal in calls to other canisters.
+* Create canisters.
+
+NOTE: Authorizing a principal as a custodian does not automatically grant the principal access to a cycles wallet. The identity assigned to the custodian role must also be assigned a cycles wallet canister identifier. For example, if you authorize the identity `+alice_custodian+` as a custodian of a cycles wallet (`+rwlgt-iiaaa-aaaaa-aaaaa-cai+`) in a local project, that user would also need to be assigned to use that wallet with the `+dfx identity set-wallet rwlgt-iiaaa-aaaaa-aaaaa-cai+` command. 
+
+[[wallet-check-balance]]
+== Check the cycle balance
+
+If you are doing local development, your cycles wallet is created when you register a new canister identifier using `+dfx canister create+` or when you register, build, and deploy a canister with `+dfx deploy+`.
+
+If you are deploying on the {IC}, you typically create your cycles wallet by converting ICP tokens to cycles, transferring the cycles to a new canister identifier, and updating the canister with the default cycles wallet WebAssembly module.
+
+After you have a cycles wallet on the local or remote network, you can use the  `+dfx wallet balance+` command or the `+wallet_balance+` method to check the current cycle balance.
+
+=== Check you cycles balance when developing locally
+
+If you are doing local development, you can use the  `+dfx wallet balance+` command to check the current cycles balance on a project-by-project basis.
+
+To check the cycles balance in a local project:
+
+. Open a terminal and navigate to the root directory of the project.
+. Start the {IC} locally by running the following command:
++
+[source,bash]
+----
+dfx start --background
+----
+. Display the cycles balance from the cycles wallet associated with the currently-selected identity by running the following command:
++
+[source,bash]
+----
+dfx wallet balance
+----
++
+The command displays output similar to the following:
++
 ....
-dfx canister --network  call f3yw6-7qaaa-aaaab-qaabq-cai wallet_create_wallet '(record { cycles = 5000000000000 : nat64; controller = principal "vpqee-nujda-46rtu-4noo7-qnxmb-zqs7g-5gvqf-4gy7t-vuprx-u2urx-gqe"})'
+78000000000000 cycles.
 ....
 
-The command returns the principal for the new wallet:
+=== Check the cycles balance on the {IC}
+
+If you have deployed a cycles wallet on the {IC} main network, you can use the  `+dfx wallet balance+` command to check the current cycles balance on the network.
 
+To check the cycles balance on the {IC}:
+
+. Open a terminal and navigate to a directory that contains a `+dfx.json+` configuration file.
+. Check your connection to the {IC} by running the following command:
```

Comment:
I could be wrong, but I think you still need to manually add non local networks to the dfx.json file...like I can ping the ic, but that doesn't mean my project is deployed to it. Might be the subnet thing though.